owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -1492,9 +1493,12 @@ private void onRemoveFileOperationFinish(RemoveFileOperation operation,
                                              RemoteOperationResult result) {
 
         if (getListOfFilesFragment().isSingleItemChecked() || result.isException() || !result.isSuccess()) {
-            showSnackMessage(
-                    ErrorMessageAdapter.getResultMessage(result, operation, getResources())
-            );
+            if(result.getCode() != ResultCode.FORBIDDEN ||
```

Comment:
Just a question, what would happen if you remove `result.getCode() != ResultCode.FORBIDDEN` and `result.getCode() == ResultCode.FORBIDDEN` conditions and do just `operation.isLastFileToRemove()`? Wouldn't it be enough?

Reply:
@davigonz `result.getCode() == ResultCode.FORBIDDEN` part allows us to show an error message if a particular file in a set of files is not allowed to be deleted (as this PR addresses the issue of just showing the snackbar once instead of multiple times for successful deletes, but I think we need to show the snackbar multiple times (each time for a forbidden delete operation)).

Reply:
@davigonz in that case the error message would just be shown once irrespective of the reason behind the message (as the issue #2471 states that the there should batched snackbar updates if an user tries delete a file he does not have permission to delete in the first place) thus the condition `result.getCode() == ResultCode.FORBIDDEN && operation.isLastFileToRemove()` is required to show the snackbar once incase of an unauthorized attempt to delete files while `result.getCoder() != ResultCode.FORBIDDEN` is required to show the snackbar for other errors that can occur like `ResultCode.FILE_NOT_FOUND` or `ResultCode.LOCAL_STORAGE_NOT_REMOVED`(in this case the snackbar will be shown for each error rather than just once.